### PR TITLE
br: add documents for  s3-multi-part-size

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add documents for tikv configuration `s3-multi-part-size`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
 + TiKV 会根据前一周期的使用情况，来调整时间戳的缓存数量。如果本地缓存使用率偏低，TiKV 会逐步降低缓存数量，直至等于 `renew-batch-min-size`。如果业务中经常出现突发的大流量写入，可以适当调大这个参数。注意这个参数是单个 tikv-server 的缓存大小，如果配置过大、而同时集群中 tikv-server 较多，会导致 TSO 消耗过快。
 + Grafana **TiKV-Raw** 面板下 **Causal timestamp** 中的 **TSO batch size** 是根据业务负载动态调整后的本地缓存数量。可以参考该监控指标值调整这个参数的大小。
 + 默认值：100
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.0 版本开始引入</span>
+
+> **注意：**
+>
+> 该配置项最初是为了解决备份期间遇到的 s3 限流问题，后来在 v6.2.0/v6.1.1/v5.4.3 中采用划分备份前缀方案后，该配置项已不再推荐使用。
```
Comment (IANTHEREAL) [inline tikv-configuration-file.md:1827]:
```suggestion
> 该配置项最初是为了解决备份期间遇到的 s3 限流问题，该问题在 v6.2.0/v6.1.1/v5.4.3 版本之后产品优化了备份数据存储的目录结构后被解决 ，因此该配置已经不推荐使用。 
```

**Review by IANTHEREAL — APPROVED** [review]

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
 + TiKV 会根据前一周期的使用情况，来调整时间戳的缓存数量。如果本地缓存使用率偏低，TiKV 会逐步降低缓存数量，直至等于 `renew-batch-min-size`。如果业务中经常出现突发的大流量写入，可以适当调大这个参数。注意这个参数是单个 tikv-server 的缓存大小，如果配置过大、而同时集群中 tikv-server 较多，会导致 TSO 消耗过快。
 + Grafana **TiKV-Raw** 面板下 **Causal timestamp** 中的 **TSO batch size** 是根据业务负载动态调整后的本地缓存数量。可以参考该监控指标值调整这个参数的大小。
 + 默认值：100
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 该配置项最初是为了缓解备份期间遇到的 s3 限流导致备份失败的问题，在 v6.2.0/v6.1.1/v5.4.3 版本之后产品优化了备份数据存储的目录结构后已经从根本上解决了该问题 ，因此该配置已经不推荐使用。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1827]:
```suggestion
> 引入该配置项是为了缓解备份期间遇到的 S3 限流导致备份失败的问题。在 v5.4.3、v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置不再推荐使用。 
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
...
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 该配置项最初是为了缓解备份期间遇到的 s3 限流导致备份失败的问题，在 v6.2.0/v6.1.1/v5.4.3 版本之后产品优化了备份数据存储的目录结构后已经从根本上解决了该问题 ，因此该配置已经不推荐使用。 
+
++ 备份阶段 s3 分块上传的块大小。
++ TiKV 备份数据到 s3 时，当备份文件大于该配置设置的大小时，会自动进行分块上传(https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_UploadPart.html)。96MB region 产生的备份文件根据压缩率的不同大约在 10MB~30MB 之间，因此该配置项也可以适当调整，来控制备份期间整体发往 s3 的请求数量。
++ 默认值：5MB
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1831]: 没看出来这个因果关系：根据压缩率的不同，96 MB Region 产生的备份文件大约在 10 MB~30 MB 之间，因此该配置项也可以适当调整，来控制备份期间整体发往 S3 的请求数量。
```suggestion
+ 备份阶段 S3 分块上传的块大小。
+ TiKV 备份数据到 S3 时，当备份文件大于该配置设置的大小时，会自动进行[分块上传](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_UploadPart.html)。根据压缩率的不同，96 MB Region 产生的备份文件大约在 10 MB~30 MB 之间，因此该配置项也可以适当调整，来控制备份期间整体发往 S3 的请求数量。
+ 默认值：5MB
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
 + TiKV 会根据前一周期的使用情况，来调整时间戳的缓存数量。如果本地缓存使用率偏低，TiKV 会逐步降低缓存数量，直至等于 `renew-batch-min-size`。如果业务中经常出现突发的大流量写入，可以适当调大这个参数。注意这个参数是单个 tikv-server 的缓存大小，如果配置过大、而同时集群中 tikv-server 较多，会导致 TSO 消耗过快。
 + Grafana **TiKV-Raw** 面板下 **Causal timestamp** 中的 **TSO batch size** 是根据业务负载动态调整后的本地缓存数量。可以参考该监控指标值调整这个参数的大小。
 + 默认值：100
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 引入该配置项是为了缓解备份期间遇到的 S3 限流导致备份失败的问题。在 v5.4.3、v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置不再推荐使用。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1827]:
```suggestion
> 引入该配置项是为了缓解备份期间遇到的 S3 限流导致备份失败的问题。在 v5.4.3、v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置目前已废弃，不再推荐使用。 
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
...
 + 默认值：100
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 引入该配置项是为了缓解备份期间遇到的 S3 限流导致备份失败的问题。在 v5.4.3、v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置不再推荐使用。 
+
++ 备份阶段 S3 分块上传的块大小。
++ TiKV 备份数据到 S3 时，当备份文件大于该配置设置的大小时，会自动进行[分块上传](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_UploadPart.html)。根据压缩率的不同，96 MB Region 产生的备份文件大约在 10 MB~30 MB 之间，因此该配置项也可以适当调整，来控制备份期间整体发往 S3 的请求数量。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1830]:
```suggestion
+ 备份阶段 S3 分块上传的块大小，可通过调整该参数来控制备份时发往 S3 的请求数量。
+ TiKV 备份数据到 S3 时，如果备份文件大于该配置项的值，会自动进行[分块上传](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_UploadPart.html)。根据压缩率的不同，96 MB Region 产生的备份文件大约在 10 MB~30 MB 之间。
```

File: tikv-configuration-file.md (hunk 4 above)
Comment (IANTHEREAL) [reply tikv-configuration-file.md:1827]: 这个就先不改了

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
 + TiKV 会根据前一周期的使用情况，来调整时间戳的缓存数量。如果本地缓存使用率偏低，TiKV 会逐步降低缓存数量，直至等于 `renew-batch-min-size`。如果业务中经常出现突发的大流量写入，可以适当调大这个参数。注意这个参数是单个 tikv-server 的缓存大小，如果配置过大、而同时集群中 tikv-server 较多，会导致 TSO 消耗过快。
 + Grafana **TiKV-Raw** 面板下 **Causal timestamp** 中的 **TSO batch size** 是根据业务负载动态调整后的本地缓存数量。可以参考该监控指标值调整这个参数的大小。
 + 默认值：100
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 引入该配置项是为了缓解备份期间遇到的 S3 限流导致备份失败的问题。在 v5.4.3、v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置目前已废弃，不再推荐使用。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1827]:
```suggestion
> 引入该配置项是为了缓解备份期间遇到的 S3 限流导致备份失败的问题。在v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置目前已废弃，不再推荐使用。 
```

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
 + TiKV 会根据前一周期的使用情况，来调整时间戳的缓存数量。如果本地缓存使用率偏低，TiKV 会逐步降低缓存数量，直至等于 `renew-batch-min-size`。如果业务中经常出现突发的大流量写入，可以适当调大这个参数。注意这个参数是单个 tikv-server 的缓存大小，如果配置过大、而同时集群中 tikv-server 较多，会导致 TSO 消耗过快。
 + Grafana **TiKV-Raw** 面板下 **Causal timestamp** 中的 **TSO batch size** 是根据业务负载动态调整后的本地缓存数量。可以参考该监控指标值调整这个参数的大小。
 + 默认值：100
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 引入该配置项是为了缓解备份期间遇到的 S3 限流导致备份失败的问题。在v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置目前已废弃，不再推荐使用。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1827]:
```suggestion
> 引入该配置项是为了解决备份期间遇到的 S3 限流导致备份失败的问题。在v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置目前已废弃，不再推荐使用。 
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
 + TiKV 会根据前一周期的使用情况，来调整时间戳的缓存数量。如果本地缓存使用率偏低，TiKV 会逐步降低缓存数量，直至等于 `renew-batch-min-size`。如果业务中经常出现突发的大流量写入，可以适当调大这个参数。注意这个参数是单个 tikv-server 的缓存大小，如果配置过大、而同时集群中 tikv-server 较多，会导致 TSO 消耗过快。
 + Grafana **TiKV-Raw** 面板下 **Causal timestamp** 中的 **TSO batch size** 是根据业务负载动态调整后的本地缓存数量。可以参考该监控指标值调整这个参数的大小。
 + 默认值：100
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 引入该配置项是为了解决备份期间遇到的 S3 限流导致备份失败的问题。在v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置目前已废弃，不再推荐使用。
```
Comment (ran-huang) [inline tikv-configuration-file.md:1827]: “在v6.1.1、v6.2.0 及以上版本”这个描述有点问题，用户不一定知道 v6.1.1 是在 6.2.0 之后发布的。

对于 dev/v6.3 用户来说这是一句多余的描述；
对 v6.2 文档，不需要加 v6.1.1；
对 v6.1 文档，不需要加 v6.2。

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
...
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 引入该配置项是为了解决备份期间遇到的 S3 限流导致备份失败的问题。在v6.1.1、v6.2.0 及以上版本，[BR 备份数据存储的目录结构进行了优化](/br/backup-and-restore-design.md#备份文件布局)，根本上解决了 S3 限流导致备份失败的问题 ，因此，该配置目前已废弃，不再推荐使用。 
+
++ 备份阶段 S3 分块上传的块大小。可通过调整该参数来控制备份时发往 S3 的请求数量。
++ TiKV 备份数据到 S3 时，如果备份文件大于该配置项的值，会自动进行[分块上传](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_UploadPart.html)。根据压缩率的不同，96 MB Region 产生的备份文件大约在 10 MB~30 MB 之间。
++ 默认值：5MB
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1831]:
```suggestion
+ 默认值：5 MB
```

File: tikv-configuration-file.md (hunk 8 above)
Comment (shichun-0415) [reply tikv-configuration-file.md:1827]: @IANTHEREAL @ran-huang 有道理，改成这样？
> 引入该配置项是为了解决备份期间遇到的 S3 限流导致备份失败的问题。目前该问题已通过[优化 BR 备份数据存储的目录](/br/backup-and-restore-design.md#备份文件布局)得到解决 ，因此，该配置目前已废弃，不再推荐使用。

然后 CP 到 6.1 和 6.2 及后续 5.4（5.4.2 发布后合并 PR）？

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1819,3 +1819,13 @@ Raft Engine 相关的配置项。
...
+
+### `s3-multi-part-size` <span class="version-mark">从 v5.3.2 版本开始引入</span>
+
+> **注意：**
+>
+> 引入该配置项是为了解决备份期间遇到的 S3 限流导致备份失败的问题。该问题已通过[优化 BR 备份数据存储的目录结构](/br/backup-and-restore-design.md#备份文件布局)得到解决。因此，该配置项自 v6.1.1 起开始废弃，不再推荐使用。
+
++ 备份阶段 S3 分块上传的块大小。可通过调整该参数来控制备份时发往 S3 的请求数量。
++ TiKV 备份数据到 S3 时，如果备份文件大于该配置项的值，会自动进行[分块上传](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_UploadPart.html)。根据压缩率的不同，96 MB Region 产生的备份文件大约在 10 MB~30 MB 之间。
++ 默认值：5 MB
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1831]:
```suggestion
+ 默认值：5MiB
```

**Review by shichun-0415 — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**